sessionctx/variable/session.go

```diff
@@ -643,7 +645,19 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 	case TiDBCurrentTS, TiDBConfig:
 		return ErrReadOnly
 	case TiDBMaxChunkSize:
-		s.MaxChunkSize = tidbOptPositiveInt32(val, DefMaxChunkSize)
+		maxChunkSize := tidbOptPositiveInt32(val, DefMaxChunkSize)
+		if s.InitChunkSize > maxChunkSize {
+			s.MaxChunkSize = mathutil.Max(s.InitChunkSize, DefMaxChunkSize)
```

Comment:
I think we can return an error here instead of a warning.

Reply:
just a previous discuss, I prefer doesn't return error, set max_limit to lower to force limit id normal.